docs/UserGuide.md

```diff
@@ -128,10 +129,61 @@ Examples:
 
 - `add n/Ben Tan p/87201223 np/90125012 a/Avenue 712 e/BenTan@gmail.com dt/2022-04-19 1500-1600`
 
-![Add](https://github.com/AY2223S1-CS2103T-T09-4/tp/blob/master/docs/images/UiAdd.png)
+![Add](images/UiAdd.png)
 
 ```yaml
 Note: Amount paid, Amount owed, Additional notes fields are to be updated via `update` command.
 ```
 
 ---
+
+### Update student details: `update`
+
+Allows the user to update the students details including:
+
+- Phone number
+- Next of Kin’s number
+- Address
+- Class Date
+- Amount paid
+- Amount owed
+- Additional notes
+
+```yaml
+Notes:
+Student’s Name must be provided to uniquely identify the student.
+Student’s Name must exist in the records beforehand.
+```
+
+1. Phone number, Next of Kin’s number, Email, Address, and Class Date follow the same convention as in Adding a student:
```

Comment:
Same with this line - perhaps this field should be named Next of Kin's phone number for clarity?

Reply:
Fix in later commit